tg/utils.py

```diff
@@ -262,3 +262,16 @@ def get_color_by_str(user: str) -> int:
         config.USERS_COLORS
     )
     return config.USERS_COLORS[index]
+
+
+def cleanup_cache() -> None:
+    if not config.KEEP_MEDIA:
+        print("Skipping cleaning cache...")
+        return
+    files_path = os.path.join(config.FILES_DIR, "files")
+    print(
```

Comment:
why you aren't using logger here? this will be printed to foreground:
```(venv) ❯ python main.py
Cleaning up cache files older than 7 days at: /home/user/.cache/tg/files
```

Reply:
When I try to use logger it does not print anything
Now I think that it's better to not show msg to user every time he launches tg.